src/osemgrep/cli_scan/Validate_subcommand.ml

```diff
@@ -139,7 +139,7 @@ let run_conf (caps : caps) (conf : conf) : Exit_code.t =
         let res = Core_runner.create_core_result metarules result_and_exn in
         (* TODO? sanity check errors below too? *)
         let OutJ.{ results; errors = _; _ } =
-          Cli_json_output.cli_output_of_core_results
+          Cli_json_output.cli_output_of_core_results ~dryrun:true
```

Comment:
here I'd appreciate feedback whether this is the right thing to do (for Validate_subcommand). Other options: (a) put a `dryrun` into the `conf` record, (b) make `dryrun` optional for `cli_output_of_core_result`.

Reply:
Validate is not fully ported, so I'm not sure what we need to do, but this looks fine for now. When I go back to finish porting validate I'll investigate.